10up/10up-experience
# Adds environment type indicator to admin bar

### Description of the Change

This adds a display to the admin bar to indicate the current environment type based on the WP_ENVIRONMENT_TYPE config variable. Most useful for increasing awareness and preventing inadvertent changes to production environment. Inspired by [Display Environment Type](https://wordpress.org/plugins/display-environment-type/) plugin.

![image](https://github.com/user-attachments/assets/0cea366d-2edd-4fcf-9014-55f73c4e7bd5)
![image](https://github.com/user-attachments/assets/5498632c-799b-4fc1-93cd-685bde34dd82)
![image](https://github.com/user-attachments/assets/b3c67a10-092c-4058-a3f0-5b26ddc4e3e2)



Closes #147 

### How to test the Change

Configure different environment types and observe that changes to indicator match.

### Changelog Entry

> Added - New feature
> Changed - Existing functionality
> Deprecated - Soon-to-be removed feature
> Removed - Feature
> Fixed - Bug fix
> Security - Vulnerability
Added indicator to admin bar to denote current environment type.

### Credits

Props @psorensen @rickalee @darylldoyle 


### Checklist:
